tests/common/include/tests/common/benchmark_find_volume.inl

```diff
@@ -32,10 +32,16 @@ auto read_detector() {
     }
     auto data_directory = std::string(env_d_d);
 
-    std::string name = "tml";
+    /*std::string name = "odd";
```

Comment:
Shall we make this method `read_detector(const std::string& detector_prifix="tml")` and then use it consistently?

Reply:
In the next PR I wrapped that up in file name structs.